src/wazuh_modules/vulnerability_scanner/indexer/template/index-template.json

```diff
@@ -239,6 +239,14 @@
                   "type": "keyword"
                 }
               }
+            },
+            "vulnerability": {
+              "properties": {
+                "under_evaluation": {
+                  "ignore_above": 1024,
+                  "type": "bool"
+                }
+              }
```

Comment:
Requirements have changed:
- https://github.com/wazuh/wazuh/issues/25482#issuecomment-2333840078

Please update the PR description as well

Reply:
Done!